src/index.js

```diff
@@ -45,8 +45,7 @@ class FontAwesome extends React.Component {
       stack,
       tag = 'span',
       ariaLabel,
-      ...props,
-    } = this.props
+      ...props } = this.props
```

Comment:
Hmm, good to know, I will fix the linting rule later, for now it's ok to ignore that error